### ISSUE TITLE
Question about the benchmark_datasets

### ISSUE THREAD

**Issue by ahandsomeperson** [mikacuy/pointnetvlad#7]
Hi, thanks for your great job! But I am trying to visualize your submaps, and the following picture is  from one of the .bin file
![图片](https://user-images.githubusercontent.com/33352875/63663506-37e23f80-c7f5-11e9-8bc5-da3a348327b2.png)
However, the max value is up to 10^38. I have no idea what is the visualization result represent for. Is there something wrong with the submap dataset? Or I miss something?

**Comment by ahandsomeperson** [comment]
oh, I just solve this problem

**Comment by KangchengLiu** [comment]
How is it solved?

**Comment by chienerh** [comment]
I also encountered the issue that the values are abnormally large.
I checked the [generate_submaps.m](https://github.com/mikacuy/pointnetvlad/blob/ec787d99627fd88c94296a7f7ad6f45621c4b95a/submap_generation/generate_submaps.m) code and found out the author is storing the bin files using double.
Previously, I was reading the bin file using np.float32. After changing the reading format to np.double or np.float64, I can read the file properly.
